README.md

```diff
@@ -58,14 +86,25 @@ The following permissions should be granted to institutional users (as well as I
                               
 ### Configuration
 
-Please refer to the [Configuration](https://github.com/folio-org/edge-inn-rach/blob/master/README.md#configuration) 
-section in the [edge-inn-reach](https://github.com/folio-org/edge-inn-reach/blob/master/README.md) documentation to see all available system properties and their default values.
+Configuration information is specified in two forms:
+1. System Properties - General configuration
+1. Properties File - Configuration specific to the desired secure store
+
+### System Properties
 
-For example, to enable HTTP compression based on `Accept-Encoding` header the `-Dresponse_compression=true` should be specified as VM option.
```

Comment:
I think providing an example of how to use the system properties when starting the jar would still be good.

Reply:
Ok, I shall put this example back and will add another example with the current system properties values used in the rancher environment as a reference to it.